build/templates/_converters.py.mako

```diff
@@ -324,3 +329,29 @@ def convert_double_each_element(numbers):
 
 
 % endif
+% for enum_name in sorted(helper.filter_codegen_enums(enums)):
+    % if helper.enum_uses_converter(enums[enum_name]):
+def ${enums[enum_name]['enum_to_converted_value_function_name']}(value):
+    try:
+        return {
+        % for enum_value in enums[enum_name]['values']:
+            enums.${enums[enum_name]['python_name']}.${enum_value['python_name']}: ${helper.get_enum_value_snippet(enum_value['converts_to_value'])},
+        % endfor
+        }[value]
+    except KeyError:
```

Comment:
after thinking about it a bit much, I think it's ok for the converter functions to return KeyError (simpler).

What to do about it is really the repsonsibility of the attribute class.

Reply:
I have removed the KeyError try-catch for the getter converter (enum_to_converted_value_function), but not the setter converter (converted_value_to_enum_function), since the Attribute class has no access to the enum conversion dictionary and would not be able to provide any helpful error message if it encounters a KeyError.